### PR TITLE
'exercise-stringer'

### PR DESCRIPTION
**Pull request by nkoblai**

### CODE REVIEW

File: go-tour/stringer.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+// TODO: Add a "String() string" method to IPAddr.
+func main() {
+	hosts := map[string]IPAddr{
+		"loopback":  {127, 0, 0, 1},
+		"googleDNS": {8, 8, 8, 8},
+	}
+	for name, ip := range hosts {
+		fmt.Printf("%v: %v\n", name, ip)
```
Comment (arbrix) [inline go-tour/stringer.go]: why do you use `%v` instead of `%s`

File: go-tour/stringer.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+)
+
+// IPAddr type represents ip-adress
+type IPAddr [4]byte
+
+func (ip IPAddr) String() string {
```
Comment (arbrix) [inline go-tour/stringer.go]: Exported function, comment is missing;
Try to find more simple solution;

**Review by arbrix — CHANGES_REQUESTED** [review]

File: go-tour/stringer.go (hunk 1 above)
Comment (nkoblai) [reply go-tour/stringer.go]: fixed

**Comment by nkoblai** [comment]
@arbrix fixed

File: go-tour/stringer.go (hunk 1 above)
Comment (arbrix) [reply go-tour/stringer.go]: make a benchmark for comparing your implementation with another one with strconv package, please

File: go-tour/stringer_test.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package stringer
+
+import (
+	"testing"
+)
+
+func BenchmarkStrConvItoa(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		for _, ip := range hosts {
```
Comment (arbrix) [inline go-tour/stringer_test.go]: this will cause additional costs for iteration, benchmark don't care about the result - it more about performance

**Review by arbrix — CHANGES_REQUESTED** [review]
change benchmark

**Review by arbrix — APPROVED** [review]
LGTM

**Pull request merged**